src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -393,15 +397,17 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
 
             if (json.Length > ScriptConstants.MaxTriggersStringLength && !_environment.IsKubernetesManagedHosting())
             {
-                // The settriggers call to the FE enforces a max request size
-                // limit. If we're over limit, revert to the minimal triggers
-                // format.
+                // The settriggers call to the FE enforces a max request size limit.
+                // If we're over limit, revert to the minimal triggers format.
                 _logger.LogWarning($"SyncTriggers payload of length '{json.Length}' exceeds max length of '{ScriptConstants.MaxTriggersStringLength}'. Reverting to minimal format.");
-                return new SyncTriggersPayload
+
+                var minimalResult = new JObject
```

Comment:
To ensure that we always return the HostID, we update the minimal format to not simply be an array. Consumers of the payload already know how to handle both object and array formats.